Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/CvarAdapter.cpp

```diff
@@ -21,10 +21,38 @@ namespace AZ::DocumentPropertyEditor
             , m_commandInvokedHandler(
                   [adapter, this](AZStd::string_view command, const AZ::ConsoleCommandContainer&, AZ::ConsoleFunctorFlags, AZ::ConsoleInvokedFrom)
                   {
-                      (void)command;
-                      if (!m_performingCommand)
+                      // look up the command being performed and rebuild its editor
```

Comment:
Well technically this after the command has been performed.
You could you add that to the comment so that readers are clear that the console command has finished executing at this point.

Reply:
thanks, fixed the comment :)